src/NzbDrone.Core/Datastore/Migration/132_add_download_client_priority.cs

```diff
@@ -0,0 +1,45 @@
+using System.Data;
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(132)]
+    public class add_download_client_priority : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("DownloadClients").AddColumn("Priority").AsInt32().WithDefaultValue(1);
+            Execute.WithConnection(InitPriorityForBackwardCompatibility);
+
+        }
+
+        private void InitPriorityForBackwardCompatibility(IDbConnection conn, IDbTransaction tran)
+        {
+            using (var cmd = conn.CreateCommand())
+            {
+                cmd.Transaction = tran;
+                cmd.CommandText = "SELECT Id FROM DownloadClients WHERE Enable = 1";
+
+                using (var reader = cmd.ExecuteReader())
+                {
+                    int i = 1;
+                    while (reader.Read())
+                    {
+                        var id = reader.GetInt32(0);
+                        
+                        using (var updateCmd = conn.CreateCommand())
+                        {
+                            updateCmd.Transaction = tran;
+                            updateCmd.CommandText = "UPDATE DownloadClients SET Priority = ? WHERE Id = ?";
```

Comment:
Should we be increasing the priority of clients of different types?

I have a torrent client and usenet client enabled, but my torrent client got priority 2 because I added it second. Instead I would expect to see both have a priority of 1.

A simple fix would be to have two counters, one for each protocol.